tuftool/src/root.rs

```diff
@@ -28,6 +28,9 @@ pub(crate) enum Command {
     Init {
         /// Path to new root.json
         path: PathBuf,
+        /// Initial metadata file version
+        #[clap(short, long)]
+        version: Option<u64>,
```

Comment:
nit: I feel like `-v` as an flag is more commonly reserved for the verbosity flag. Without context, `-v 5` reads as "set log level to trace". Can we just have `long` and no short flag option?

Reply:
Yeah, that's fair. I did kind of wonder about that myself. Even `--version` can be convention for printing out the version of the tool. I was thinking about `--init-version`, `--start-version`, and a few others, but didn't really like any of the options I was coming up with. I think if we just go with the long `--version`, it's at least documented in the help output and should (hopefully) be clear from its context.

Reply:
Changed to just have the `long` form. New help output:

```
Create a new root.json metadata file

USAGE:
    tuftool root init <PATH>

ARGS:
    <PATH>    Path to new root.json

OPTIONS:
    -h, --help                 Print help information
        --version <VERSION>    Initial metadata file version
```